packages/mwp-app-server/src/util/index.js

```diff
@@ -27,6 +27,13 @@ export function onRequestExtension(request, h) {
 	return h.continue;
 }
 
+export function onPreResponseExtension(request, h) {
+	// response header corresponding to the application version tag generated by CI
```

Comment:
since this function doesn't control _how_ the env var is set, maybe just say that a version tag can be passed in as an env var and returned as a header